lib/internal/util.js

```diff
@@ -10,7 +18,7 @@ exports.printDeprecationMessage = function(msg, warned) {
   if (process.throwDeprecation)
     throw new Error(msg);
   else if (process.traceDeprecation)
-    console.trace(msg);
+    console.trace(msg.replace(new RegExp(`^\\(${prefix}\\) `), ''));
```

Comment:
@silverwind Actually, the ES6 proposal had [`re` Quasi literal function](http://wiki.ecmascript.org/doku.php?id=harmony:quasis#flexible_literal_syntax) though :-)